code/modules/projectiles/guns/energy/pulse.dm

```diff
@@ -33,41 +37,47 @@
 	. = ..()
 
 /obj/item/gun/energy/pulse/loyalpin
-	pin = /obj/item/firing_pin/implant/mindshield
+	pin = /obj/item/firing_pin
 
 /obj/item/gun/energy/pulse/carbine
 	name = "pulse carbine"
-	desc = "A compact variant of the pulse rifle with less firepower but easier storage."
-	w_class = WEIGHT_CLASS_NORMAL
-	slot_flags = ITEM_SLOT_BELT
+	desc = "The next-generation rifle of nanotrasen, however the logistical issues of making it the standard has left it only used by specialist LP and ERT teams."
```

Comment:
Reworded, grammar edits.

"A next-generation pulse weapon for Nanotrasen's security forces. High production costs and logistical issues have limited its deployment to specialist Loss Prevention and Emergency Response units."